blivet/util.py

```diff
@@ -1117,4 +1117,4 @@ def detect_virt():
     except (safe_dbus.DBusCallError, safe_dbus.DBusPropertyError):
         vm = None
 
-    return vm in ('qemu', 'kvm')
+    return vm[0] in ('qemu', 'kvm')
```

Comment:
If we catch an exception above `vm` will be None and this will cause a traceback.

Reply:
Fixed.